File: fixtures/node_output/get_with_url_encoding.js
var request = require('request');

var dataString = 'data=1&text=%D2%D4';

var options = {
    url: 'http://example.com',
    method: 'POST',
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/rust_output/get_with_url_encoding.rs
extern crate reqwest;

fn main() -> Result<(), reqwest::Error> {

    let res = reqwest::Client::new()
        .post("http://example.com")
        .body("data=1&text=%D2%D4")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}
